drink/src/session.rs

```diff
@@ -336,11 +357,21 @@ impl<R: Runtime> Session<R> {
         self.call_results.last()
     }
 
-    /// Returns the last value returned from calling a contract.
-    pub fn last_call_return(&self) -> Option<Value> {
+    /// Returns the last value (in the encoded form) returned from calling a contract.
+    ///
+    /// Returns `None` if there has been no call yet.
+    pub fn last_call_raw_return(&self) -> Option<Vec<u8>> {
         self.call_returns.last().cloned()
     }
 
+    /// Returns the last value (in the encoded form) returned from calling a contract.
```

Comment:
Shouldn’t the docstring say „decoded” here?

Reply:
fixed